tests/integration.rs

```diff
@@ -743,3 +745,65 @@ fn test_expire() -> Result<()> {
 
     Ok(())
 }
+
+#[test]
+fn test_defrag() -> Result<()> {
+    let port: u16 = 6503;
+    let _guards = vec![start_redis_server_with_module("data_type", port)
```

Comment:
So this is just one guard, not a vector of multiple?

Reply:
I actually not sure why it is a vec, but this is how it is done on all the other tests, will leave to change it on another PR.

Reply:
No problem.